.github/workflows/test_pull_requests.yml

```diff
@@ -0,0 +1,120 @@
+# Our minimal suite of tests that run on each pull request
+name: Quick PR Tests
```

Comment:
Its just so many letters on the dialog otherwise :-) 
![image](https://user-images.githubusercontent.com/6531703/103380846-ef006200-4a9e-11eb-90c6-1fb8cb1b8bc1.png)

Reply:
agreed. I'll shorten them all, but I'd like to leave in test (since we also have docs, bundle, etc...) so how about `PR Test`, `Full Test`, `--pre Test` ... and I can shorten `Code Quality` too

Reply:
those are fine names, i was just trying to trim down a little, don't have strong feelings